src/web/client/extension.ts

```diff
@@ -254,6 +258,125 @@ export function processWillSaveDocument(context: vscode.ExtensionContext) {
     );
 }
 
+let copresenceWorker: Worker;
```

Comment:
this needs to shift as retrun parameter of function processOpenActiveTextEditor(), correct?

Reply:
tried didn't worked! so keeping it as it is